lms/templates/main.html

```diff
@@ -148,8 +148,8 @@
 
   <meta name="openedx-release-line" content="${RELEASE_LINE}" />
 
-<% ga_acct = static.get_value("GOOGLE_ANALYTICS_ACCOUNT", settings.GOOGLE_ANALYTICS_ACCOUNT) %>
-% if ga_acct:
+  <% ga_acct = static.get_value("GOOGLE_ANALYTICS_ACCOUNT", settings.GOOGLE_ANALYTICS_ACCOUNT) %>
```

Comment:
As mentioned in [a previous comment](https://github.com/edx/edx-platform/pull/28411#discussion_r687197758), would you please revert the indentation fixes for the code you didn't add/isn't related to this pull request? 

I'll avoid commenting on each of those lines, but if you would prefer I do so, let me know.